backend/functions/src/scheduled/drizzle-liquidity.ts

```diff
@@ -62,31 +70,30 @@ const drizzleMarket = async (contractId: string) => {
     const amount = subsidyPool <= 1 ? subsidyPool : r * v * subsidyPool
 
     if (contract.mechanism === 'cpmm-multi-1') {
-      const answersSnap = await trans.get(
-        firestore.collection(`contracts/${contractId}/answersCpmm`)
-      )
-      const answers = answersSnap.docs.map((doc) => doc.data() as Answer)
+      if (!contract.shouldAnswersSumToOne) return // will be drizzled by drizzleAnswer
```

Comment:
Not true. If you subsidize independent multi from the boost button, it does it on the whole market rather than specific answers. Then this code is supposed to drizzle it evenly among answers